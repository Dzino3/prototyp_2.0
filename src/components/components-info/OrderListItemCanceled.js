import { t } from 'i18next'
import React from 'react'

function OrderListItemCanceled({item}) {
  return (
    <div className='orderListItemCanceled'>
        <div className='block'>
            <p className='p2'>{t('listItemCLabel1')} </p>
            <p>{item.RDD}</p>
            <p className='p2'>{t('listItemCLabel2')} </p>
            <p>{item.REQ_SHIP_DATE}</p>
            <p className='p2'>{t('listItemCLabel3')} </p>
            <p>{item.ORDER_ENTRY_DATE}</p>
        </div>
        <div className='block'>
            <p className='p2'>{t('listItemCLabel4')} </p>
            <p>{item.DATE_OF_CANCELLATION}</p>
            <p className='p2'>{t('listItemCLabel5')} </p>
            <p>{item.CANCELLATION_MONTH}</p>
            <p className='p2'>{t('listItemCLabel6')} </p>
            <p>{item.REASON}</p>
        </div>
    </div>
  )
}

export default OrderListItemCanceled
