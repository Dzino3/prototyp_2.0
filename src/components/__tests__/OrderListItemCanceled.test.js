import React from "react";
import renderer from 'react-test-renderer';
import OrderListItemCanceled from '../components-info/OrderListItemCanceled';

describe(`Rendering OrderListItemCanceled`, () => {

    test('should render OrderListItemCanceled with item CANCELED', function () {
        let itemX = {
            item: "CANCELED"
        }
        let tree = renderer.create(<OrderListItemCanceled item={itemX}></OrderListItemCanceled>);

        expect(tree.toJSON()).toMatchSnapshot();
    });

    test('should render OrderListItemCanceled with no item', function () {
        let itemX = {}
        let tree = renderer.create(<OrderListItemCanceled item={itemX}></OrderListItemCanceled>);

        expect(tree.toJSON()).toMatchSnapshot();
    });
})