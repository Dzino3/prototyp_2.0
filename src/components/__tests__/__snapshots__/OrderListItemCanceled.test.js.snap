// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Rendering OrderListItemCanceled should render OrderListItemCanceled with item CANCELED 1`] = `
<div
  className="progressBarhelper"
>
  <div
    className="orderListItemCanceled"
  >
    <div
      className="block"
    >
      <p
        className="p2"
      >
         
      </p>
      <p />
      <p
        className="p2"
      >
         
      </p>
      <p />
    </div>
    <div
      className="block"
    >
      <p
        className="p2"
      >
         
      </p>
      <p />
      <p
        className="p2"
      >
         
      </p>
      <p />
    </div>
    <div
      className="block"
    >
      <p
        className="p2"
      >
         
      </p>
      <p />
      <p
        className="p2"
      >
         
      </p>
      <p />
    </div>
  </div>
  <div
    className="progressBarImgC"
  />
</div>
`;

exports[`Rendering OrderListItemCanceled should render OrderListItemCanceled with no item 1`] = `
<div
  className="progressBarhelper"
>
  <div
    className="orderListItemCanceled"
  >
    <div
      className="block"
    >
      <p
        className="p2"
      >
         
      </p>
      <p />
      <p
        className="p2"
      >
         
      </p>
      <p />
    </div>
    <div
      className="block"
    >
      <p
        className="p2"
      >
         
      </p>
      <p />
      <p
        className="p2"
      >
         
      </p>
      <p />
    </div>
    <div
      className="block"
    >
      <p
        className="p2"
      >
         
      </p>
      <p />
      <p
        className="p2"
      >
         
      </p>
      <p />
    </div>
  </div>
  <div
    className="progressBarImgC"
  />
</div>
`;
